tests/MonoGame.Extended.Content.Pipeline.Tests.Tiled/KNI.Extended.Content.Pipeline.Tests.Tiled.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <PackageReference Remove="MonoGame.Framework.DesktopGL" />
+
+    <PackageReference Include="nkast.Xna.Framework" Version="3.12.9001" />
+    <PackageReference Include="nkast.Xna.Framework.Content" Version="3.12.9001" />
+    <PackageReference Include="nkast.Xna.Framework.Graphics" Version="3.12.9001" />
+    <PackageReference Include="nkast.Xna.Framework.Input" Version="3.12.9001" />
+    <PackageReference Include="nkast.Xna.Framework.Game" Version="3.12.9001" />
+    <PackageReference Include="nkast.Xna.Framework.Content.Pipeline" Version="3.12.9001" />
```

Comment:
See previous comment about moving references to the `Directory.Build.props`

You'll need to leave the content pipeline references here since they are specific to this project and not all projects.